client/src/app/pages/migration-waves/components/ticket-issue.tsx

```diff
@@ -1,25 +1,40 @@
 import React from "react";
-import { Text } from "@patternfly/react-core";
+import { Button, Text, TextVariants } from "@patternfly/react-core";
 
 import { Ticket } from "@app/api/models";
-import { useTrackerTypesByProjectId } from "@app/queries/trackers";
+import { useTranslation } from "react-i18next";
+import { UnlinkIcon } from "@patternfly/react-icons";
+import { useDeleteTicketMutation } from "@app/queries/migration-waves";
 
 export interface ITicketIssueProps {
   ticket?: Ticket;
 }
 
 export const TicketIssue: React.FC<ITicketIssueProps> = ({ ticket }) => {
-  const useTicketIssue = () => {
-    const types = useTrackerTypesByProjectId(
-      ticket?.tracker?.name,
-      ticket?.parent
-    );
-    const type = types.find((kind) => kind.id === ticket?.kind);
-    if (type) return type.name;
-    return "";
-  };
+  const { t } = useTranslation();
+  const { mutate: deleteTicket } = useDeleteTicketMutation();
 
-  const ticketIssue = useTicketIssue();
-
-  return <Text>{ticketIssue}</Text>;
+  return (
+    <>
+      <Text
+        component={TextVariants.p}
+        className="pf-v5-u-color-200 pf-v5-u-font-weight-light"
+      >
+        {ticket ? (
+          <a href={ticket.link} target="_">
+            {ticket?.link}
+          </a>
+        ) : (
+          t("terms.unassigned")
+        )}
+        {ticket?.id && (
+          <Button
+            variant="link"
+            icon={<UnlinkIcon />}
+            onClick={() => deleteTicket(ticket.id)}
+          />
+        )}
```

Comment:
Would it be better to put the unlink icon button in the wave-status-table by the trash icon button?

Reply:
Not sure only because this is how it was mocked up. I think I do like the idea of moving to the actions column, though.

Reply:
Looks like the mock-up is sufficiently ambiguous on this one:
![image](https://github.com/konveyor/tackle2-ui/assets/3985964/2ef341b6-49d2-42f1-bd0a-e1177e7216cc)

Reply:
100%!